### PR TITLE
alignedProcessFieldTitle() code improvements

### PR DESCRIPTION
**Pull request by Explorer09**
Two minor improvements to the alignedProcessFieldTitle() subroutine.
See the commit descriptions for details.

### CODE REVIEW

**Comment by BenBE** [comment]
Another thing I'd like to add to this would be consistent handling of the trailing space for the sort indicator. Currently for auto-sized columns it's added by the software, whereas for statically sized columns it's included in the static string. Would be nice to have this uniformly handled independent of the auto-size property.

File: ProcessList.c [hunk 1]
```diff
@@ -82,40 +82,41 @@ void ProcessList_setPanel(ProcessList* this, Panel* panel) {
...
    int width = column->width;
    if (!width || abs(width) > DYNAMIC_MAX_COLUMN_WIDTH)
       width = DYNAMIC_DEFAULT_COLUMN_WIDTH;
-   xSnprintf(titleBuffer, sizeof(titleBuffer), "%*s", width, column->heading);
+   xSnprintf(titleBuffer, titleBufferSize, "%*s", width, column->heading);
    return titleBuffer;
 }
 
 static const char* alignedProcessFieldTitle(const ProcessList* this, ProcessField field) {
+   static char titleBuffer[UINT8_MAX + sizeof(" ")];
```
[the remark this answers is not in the document]
Comment (Explorer09) [reply ProcessList.c:97]: The maximum width of a process field in general is UINT8_MAX. It's hinted in the old code. The buffer is allocated statically so it won't hurt much.

[the remark this answers is not in the document]
Comment (BenBE) [reply ProcessList.c:97]: Ah, I interpreted that buffer to hold some number, thus I was confused here …

**Comment by BenBE** [comment]
@Explorer09 Please rebase and cater to that one open remark re `char[]` -> `char*`.

**Review by BenBE — APPROVED** [review]

**Pull request merged**